week3/helpMenu.txt

```diff
@@ -0,0 +1,63 @@
+---------------------                                                                            ||
+$ node index.js list >>>>>>> To get a list of all tasks.                                         ||
+---------------------        Arguments after list will be ignored.                               ||
+                                                                                                 ||
+-----------------------------------                                                              ||
+$ node index.js add "a task to do" >>>>>>>> To add a task to the list, all the words behind      ||
+ ----------------------------------         "add" are entered as 1 todo item                     ||
+                                            to the list (node index.js add "Buy groceries"),     ||                    
+                                            and the task should be surrounded with quotes.       ||       
+                                                                                                 ||
+------------------------                                                                         ||
+$ node index remove {n} >>>>>>>>>> To remove task number {n} from the list.                      ||
+------------------------                                                                         ||
+                                                                                                 ||
+----------------------                                                                           ||
+$ node index.js reset >>>>>>>>> To delete all tasks and reset the list.                          ||
+----------------------          Arguments after reset will be ignored.                           ||
+                                                                                                 ||
+------------------------------------------                                                       ||
+$ node index.js update {n} "updated task" >>>>>>>>> To update task number {n} with new task.     ||
+------------------------------------------                                                       ||
+                                                                                                 ||
+---------------------                                                                            ||
+$ node index.js help >>>>>>>>> To display the help menu, this instruction menu.                  ||
+---------------------                                                                            ||
+                                                                                                 ||
+                                                                                                 || 
+                                                                                                 ||
+==================================================================================================
```

Comment:
Thanks for writing a comprehensive 'help' file, it makes it A LOT easier to review and give feedback!